modules/roles_profiles/manifests/profiles/gecko_t_osx_1014_generic_worker_multiuser.pp

```diff
@@ -46,8 +46,8 @@
                 quarantine_access_token   => $quarantine_access_token,
                 bugzilla_api_key          => $bugzilla_api_key,
                 task_dir                  => '/Users',
-                generic_worker_version    => 'v16.4.0',
-                generic_worker_sha256     => 'eaf18131ff7f7e28810554719c71e5b658bec374d03ca469c5b9af272920e331',
+                generic_worker_version    => 'v16.5.2',
+                generic_worker_sha256     => '7bd47da57aae65f120d89e8d70fb0a1f66762945994e0909d31eac6d63122046',
```

Comment:
https://tools.taskcluster.net/groups/FEKcrpGcQ4uRXVJ6-cl9PQ/tasks/JCtMJWbbRz2xVWgkpbdWjg/runs/0/logs/public%2Flogs%2Flive_backing.log#L14 is from a couple of hours ago and shows `t-mojave-r7-244` is in the staging pool (`releng-hardware/gecko-t-osx-1014-staging`) but running generic-worker 15.1.4.